lib/rubocop/cop/rails/before_destroy.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for a `before_destroy` callback that don't
```

Comment:
I think this should read either:
"This cop looks for `before_destroy` callbacks that don't"
or
"This cop looks for a `before_destroy` callback that doesn't"

Reply:
Thank you so much for reviewing!
I revise a sentence to the plural. e5524b2

(I'm sorry for using the wrong grammar. I've been using [Grammarly: Free Online Writing Assistant](https://www.grammarly.com/) to check the English language, but couldn't check it. If you have any other ideas, please let me know!)